node/service/src/parachains_db/mod.rs

```diff
@@ -36,12 +36,25 @@ pub(crate) mod columns {
 
 	pub mod v2 {
 		pub const NUM_COLUMNS: u32 = 6;
+		// pub const COL_AVAILABILITY_DATA: u32 = 0;
```

Comment:
commented code needs to be removed.

Reply:
@sandreim does this mean that any columns used from  `v2::` should now be used from `v3::`?

E.g.
https://github.com/paritytech/polkadot/blob/39ae1910287c2de0fde8b18ac04a1a4623d77737/node/service/src/parachains_db/upgrade.rs#L302
will become
`super::columns::v3::COL_DISPUTE_COORDINATOR_DATA`

And second question - I see there is no `ORDERED_COL` in the old versions. Should I also keep this only in the latest one?

Reply:
Yeah, I think we should follow the current pattern, keep current code references in latest version, prev versions stuff only for testing.

Reply:
Fixed